Appraisals

```diff
@@ -1,50 +1,12 @@
-appraise "rails-3-2" do
-  group :mysql do
-    gem "mysql2", "~> 0.3.21", platforms: [:ruby]
+{
+  '3.2' => '3.2.0',
+  '4.1' => '4.1.0',
+  '4.2' => '4.2.0',
+  '5.0' => '5.0.0',
+  '5.1' => '5.1.0',
+  '6.0' => '6.0.0.rc1'
+}.each do |rails, version|
+  appraise "rails-#{rails}" do
+    gem "rails", "~> #{version}"
   end
-  gem "activerecord", "~> 3.2.22.2"
-  gem "rake", "~> 12.2.0", platforms: [:ruby_19]
-  group :test do
-    gem "after_commit_exception_notification"
-  end
-end
-
-appraise "rails-4-1" do
-  group :mysql do
-    gem "mysql2", "~> 0.3.21", platforms: [:ruby]
-  end
-  gem "activerecord", "~> 4.1.16"
-  gem "rake", "~> 12.2.0", platforms: [:ruby_19]
-  group :test do
-    gem "after_commit_exception_notification"
```

Comment:
Can you check that this simplification doesn't skip anything out? :)

Reply:
I double checked it and everything is in place

Reply:
Thanks for that. I don't get how a simple `gem "rails"...` extrapolates to including all the other gems for testing under specific rails versions like `after_commit_exception_notification`. I did test it locally and it does generate as you say, is that just because the existing lock files satisfy the decencies. Can you double-check deleting the `gemfiles` directory and regenerating the files from scratch to see if they change?

Reply:
right, my bad, I just added the new `gemfiles`

Reply:
Thanks @jtapia, unfortunately you can't edit the gemfiles directly as they're generated from the `Appraisal` file, so likely you'll have to revert that whole file and just add in your rails 6 part, then run `appraisal generate` to generate the new files.

Reply:
Check out https://github.com/thoughtbot/appraisal#commands if you haven't already, though I suspect you have :)

Reply:
done